docs/source/reference/kubernetes/kubernetes-setup.rst

```diff
@@ -169,9 +169,12 @@ Setting up GPU support
 ~~~~~~~~~~~~~~~~~~~~~~
 If your Kubernetes cluster has Nvidia GPUs, ensure that:
 
-1. The Nvidia device plugin is installed (i.e., ``nvidia.com/gpu`` resource is available on each node).
+1. The Nvidia GPU operator is installed (i.e., ``nvidia.com/gpu`` resource is available on each node) and is ``nvidia`` is set as the default runtime for your container engine. See `Nvidia's installation guide <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#install-nvidia-gpu-operator>`_ for more details.
```

Comment:
```suggestion
1. The Nvidia GPU operator is installed (i.e., ``nvidia.com/gpu`` resource is available on each node) and ``nvidia`` is set as the default runtime for your container engine. See `Nvidia's installation guide <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#install-nvidia-gpu-operator>`_ for more details.
```
Is it possible to give a quick tip on how to check the second condition 
> ``nvidia`` is set as the default runtime for your container engine.

?

Reply:
I checked, unfortunately it depends on the specific container runtime in use (and can be overriden by k8s). Best is to check by running a small job, added a note + k8s yaml on that.